iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -141,7 +141,7 @@ class RouDi
 
   private:
     std::thread m_processManagementThread;
-    std::thread m_processMQThread;
+    std::thread m_processRuntimeMessagesThread;
```

Comment:
This is confusing. Why does something called `IpcThreadStart` control something called `m_processRuntimeMessagesThread`?

Reply:
Ups :blush: Missed the `IpcThreadStart`. Fixed.